src/journal/Settings.h

```diff
@@ -12,7 +12,7 @@ struct Settings {
   double commit_interval = 5;         ///< commit position throttle (in secs)
   uint64_t max_payload_bytes = 0;     ///< 0 implies object size limit
   int max_concurrent_object_sets = 0; ///< 0 implies no limit
-  std::set<std::string> whitelisted_laggy_clients;
+  std::set<std::string> allowlisted_laggy_clients;
```

Comment:
Nit:  "allowlisted" is pretty awkward, perhaps:
```suggestion
  std::set<std::string> ignored_laggy_clients;
```

Reply:
How about allowed_laggy_clients like ALLOWED_RESTORE_SOURCES below?

Reply:
@markhpc I have incorporated @dillaman's suggestion. If you have a strong opinion about this, I can change it again. Either sounds fine to me

Reply:
@neha-ojha mostly I just want to keep the terminology consistent and semantically accurate.  IE do we ignore laggy clients or do we allow laggy clients?  I think they have different meanings, but depending on what the code does one might be more accurate than the other.

Reply:
I think either is honestly fine. The laggy clients are being ignored / allowed to exist.

Reply:
@dillaman when I see ignored the first thought that comes to mind is "we are ignoring these clients because they are laggy" vs for allowed: "we are allowing these clients despite them being laggy".  It's entirely possible I'm reading this differently than others would though, so please go with whatever makes the most sense to you!